VMEncryption/main/check_util.py

```diff
@@ -223,11 +224,20 @@ def validate_lvm_os(self, public_settings):
         if detected:
             raise Exception("LVM OS disk layout does not satisfy prerequisites ( see https://aka.ms/adelvm )")
 
+    def validate_vfat(self):
+        """ Check for vfat module using modprobe and raise exception if not found """
+        try:
+            executor = CommandExecutor(self.logger)
+            executor.Execute("modprobe vfat", True)
```

Comment:
Does this command remain the same for all distros?
1) Do Ubuntu and RedHat both have the command "modprobe"?
2) Is the module called "vfat" in both distros?

Reply:
Yes, good question.  Testing showed that the command is available across all currently supported image types (RedHat,CentOS,Ubuntu,SUSE) and the vfat module was found in each of those as well. We are using modprobe elsewhere in the extension.